examples/image-resize/app/routes/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { Image } from '~/components/image'
+
+export default function Index() {
+  return (
+    <div className="container">
+      <h1 className="mt-12 text-4xl">Cover</h1>
+      <div className="flex gap-4 items-center overflow-x-scroll">
```

Comment:
Could use inline styles for this example.

Reply:
I'll fix this together with the tailwind removal